simple-chat-front/src/components/Message/Message.jsx

```diff
@@ -2,11 +2,12 @@ import React from 'react';
 import './Message-Style.css'
 
 
-const Message = ({ content, imSender }) => {
+const Message = ({ message, imSender, createdAt, userName }) => {
     return (
+
         <div className={`message ${imSender === true ? "message--me" : null}`}>
-            <span className='message__text'>{content}</span>
-            <span className='message__time'>8:00</span>
+            <span className='message__text'>{message}</span>
+            <span className='message__time'>{createdAt}</span>
```

Comment:
This library is pretty good for parsing date: https://github.com/PseudoMera/date-parser

Reply:
😎👍